.nycrc

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "@ethereumjs/config-nyc",
+  "include": [
+    "dist/**/*.js"
+  ],
+  "extension": [
+    ".js"
+  ],
+  "source-map": false
+}
```

Comment:
Side note: we should from time to time re-evaluate a bit which settings which are manually added to the config files make generally sense and can be shifted to the generic config files.